Tools/Bluetooth/README.md

```diff
@@ -0,0 +1,62 @@
+# Bluetooth Tools
+These python tools are used for Bluetooth development and debugging.
+
+## BLE_hci
+Host Controller Interface for controlling Bluetooth Controller devices. Uses a serial port and the HCI interface to send commands and receive events.
+
+```
```

Comment:
Can we double check this is the current help text?

Reply:
"HCI Tool used to control a Bluetooth Controller through a serial port." is the current help text in the main branch python file, but no README is present there

Reply:
If you run with the -h parameter and send the help command then we get this help text.

``` bash
python3 BLE_hci.py -h
python3 BLE_hci.py
help
```